### PR TITLE
[filestorage] - update man page for strings

### PR DESCRIPTION
**Pull request by VihasMakwana**
This is a simple fix to make `check-links` workflow green.
It returns `403 FORBIDDEN` while accessing https://linux.die.net/man/1/strings. Switching to https://man7.org/linux/man-pages/man1/strings.1.html fixes it

### CODE REVIEW

**Comment by VihasMakwana** [comment]
@open-telemetry/collector-approvers can I get quick approval here, as it blockes https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/34985?

**Review by songy23 — APPROVED** [review]

**Review by crobert-1 — APPROVED** [review]

**Pull request merged**
